app/assets/javascripts/discourse/app/controllers/full-page-search.js

```diff
@@ -229,6 +231,11 @@ export default Controller.extend({
     }
   },
 
+  @discourseComputed("currentUser.use_experimental_topic_bulk_actions")
+  useNewBulkActions() {
+    return this.currentUser?.use_experimental_topic_bulk_actions;
```

Comment:
looks like this controller is missing an explicit `currentUser` service injection - could you add it?

Reply:
Sure, will add :+1: